src/support/azext_support/aaz/latest/support/in_subscription/communication/_list.py

```diff
@@ -15,7 +15,7 @@
     "support in-subscription communication list",
 )
 class List(AAZCommand):
-    """List all communications (attachments not included) for a support ticket. <br/></br> You can also filter support ticket communications by _CreatedDate_ or _CommunicationType_ using the $filter parameter. The only type of communication supported today is _Web_. Output will be a paged result with _nextLink_, using which you can retrieve the next set of Communication results. <br/><br/>Support ticket data is available for 18 months after ticket creation. If a ticket was created more than 18 months ago, a request for data might cause an error.
+    """List all communications (attachments not included) for a support ticket.  You can also filter support ticket communications by CreatedDate or CommunicationType using the --filter parameter. The only type of communication supported today is Web. Output will be a paged result with nextLink, using which you can retrieve the next set of Communication results. Support ticket data is available for 18 months after ticket creation. If a ticket was created more than 18 months ago, a request for data might cause an error.
 
```

Comment:
Remove <br/> would cause word in web page too dense. Change it to legal html tag <br> would be proper.

Reply:
They are legal html tags and I'll deal with it from azdev tool.